doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -315,6 +315,44 @@ YAML::
             # Default: all.
             #types: [a, aaaa, cname, mx, ns, ptr, txt]
 
+Fileinfo
+~~~~~~~~
+
+Fileinfo records are logged when files are seen on network traffic. The log
+contains information about the file, such as its name, size, and
+hashes.
+
+The type of the file can also be determined by doing an analysis of the beginning of the content
+of the file. This is done by using libmagic and/or libmimetype. Magic is slower than mimetype, but 
+it can provide more information about the file type. At the same time, magic output is less predictive than
+mimetype one that follows a standard.
+
+If you want to log the file type, you can enable either or both of these options.
+
+YAML::
+
+        - files:
+            force-magic: no   # force logging magic on all logged files
+            force-mimetype: no # force logging mime type on all logged files
+            # force logging of checksums, available hash functions are md5,
+            # sha1 and sha256
+            #force-hash: [md5]
+
+The identification by magic is using a file available on the system. The file can be set by using a
+dedicated configuration option:
+
+YAML::
+
+magic-file: /usr/share/file/magic
+
+The mimetype analysis is using the system's MIME type files if the bundled GPL data is
+not included at build time. In that case, the used files have the same syntax as the one
+used by magic but they are auto discovered using predefined path and paths
```

Comment:
so this means we have another keyword that is unpredictable to rule writers? System installed magic can have quite a bit of variance as we have experienced in the past.

Reply:
Yes, if you choose to remove the bundled GPL database at build time then it is not predictable. But by default the build will produce reproducible results.

Reply:
That is unfortunate. So if the gpl data isn't used, the regular magic data is used? How does it work, does it fall back to libmagic or does the crate support the magic format?

Reply:
The crate supports the magic format but it does its own matching. It loads and concatenate various magic file present on the system if data is not embedded. Output is always in the MIME type form.

Reply:
I think we can not do it without modification of the crate. What if I try to propose an option to load a specific file to the crate developers and we include that if it reaches upstream ?

Reply:
Sounds to me like it would be a good feature for such a crate.

Reply:
I've opened a PR on the crate: https://github.com/mbrubeck/tree_magic/pull/27

Reply:
And it has been merged. Huge thanks to @mbrubeck! I will propose a new PR once the \tree_magic_mini` crate has a new version released.